pkg/resource/deploy/source_eval.go

```diff
@@ -732,6 +791,8 @@ func (rm *resmon) SupportsFeature(ctx context.Context,
 		hasSupport = true
 	case "deletedWith":
 		hasSupport = true
+	case "transformations":
```

Comment:
I thought we were going to change this to "transforms"?

Reply:
Yup, done.